packages/design-system/src/components/N8nSelect/Select.vue

```diff
@@ -15,7 +107,7 @@
 			:model-value="modelValue"
 			:size="computedSize"
 			:class="$style[classes]"
-			:popper-class="popperClass"
```

Comment:
There was a bug here. The computed is named `popperClasses` but here we use `popperClass`, which is the name of the prop. I changed this to use the computed. Should we instead keep this as is? I.e. use the prop directly instead of the computed, which adds the `limitPopperWidth` class